tests/unittests/tensorsTest.cpp

```diff
@@ -62,6 +62,18 @@ TEST(Tensor, randomizeInt) {
   }
 }
 
+TEST(Tensor, randomizeFloat16) {
+  PseudoRNG PRNG;
+  Tensor T(ElemKind::Float16Ty, {10, 10});
+  auto H = T.getHandle<float16_t>();
+  H.randomize(-50, 50, PRNG);
+
+  // Check that all of the numbers fall in the range -50 to 50.
+  for (size_t i = 0, e = H.size(); i < e; i++) {
+    EXPECT_NEAR(H.raw(0), 0, 50);
```

Comment:
`H.raw(i)`

Reply:
😱
Nice catch!

Reply:
I did a copy paste from the previous test and that one is wrong too!!